node_modules/oae-principals/lib/api.user.js

```diff
@@ -354,9 +375,6 @@ var importUsers = module.exports.importUsers = function(ctx, tenantAlias, userCS
                             if (!user.publicAlias || user.publicAlias === externalId) {
                                 update['publicAlias'] = displayName;
                             }
-                            if (!user.email) {
-                                update['email'] = opts.email;
-                            }
```

Comment:
Why wouldn't we do this anymore. i.e., update a user's email address with what is in the update CSV if they don't currently have one?